website/content/en/preview/upgrading/compatibility.md

```diff
@@ -15,9 +15,9 @@ Before you begin upgrading Karpenter, consider Karpenter compatibility issues re
 
 [comment]: <> (the content below is generated from hack/docs/compataiblitymetrix_gen_docs.go)
 
-| KUBERNETES |   1.23   |   1.24   |   1.25   |   1.26   |   1.27   |   1.28   |    1.29    |
```

Comment:
Looks like we're reverting this #5842 since they only updated the docs but didn't update the script which generates this table. Maybe manually revert here and we can cut another PR to update the generation script?

Reply:
Nevermind, looks like the change was made but it's still being reverted by docgen for preview. Going to fix manually for this PR and follow-up.